src/main/cliapp/src/service/TableStateService.js

```diff
@@ -155,13 +155,33 @@ const modTableSettings = {
 			tableKeyName: "DiseaseAnnotations",
 			tableSettingsKeyName: "DiseaseAnnotationsTableSettings"
 		}
+	},
+	DefaultStaff: {
+		DiseaseAnnotations: {
+			page: 0,
+			first: 0,
+			rows: 50,
+			multiSortMeta: [],
+			selectedColumnNames: [
+				"Unique ID", "MOD Annotation ID", "Subject", "Disease Relation", "Negated", "Disease", "Reference", "Evidence Code", "With",
+				"Related Notes", "Experiments" ,"Experimental Conditions", "Genetic Sex" ,"Disease Qualifiers", "SGD Strain Background" , "Annotation Type" ,
+				"Genetic Modifier Relation" , "Genetic Modifier" , "Inferred Gene", "Asserted Genes", "Inferred Allele", "Asserted Allele", "Data Provider",
+				"Secondary Data Provider", "Updated By", "Date Updated", "Created By", "Date Created", "Internal", "Obsolete"
+			],
+			filters: {},
+			isFirst: false,
+			tableKeyName: "DiseaseAnnotations",
+			tableSettingsKeyName: "DiseaseAnnotationsTableSettings"
+		}
 	}
 }
 
+const modStaffList = ['RGDStaff','SGDStaff','WBStaff','FBStaff','ZFINStaff','MGIStaff'];
+
 export function getModTableState(table){
 	const oktaToken = JSON.parse(localStorage.getItem('okta-token-storage'));
 	const mod = oktaToken?.accessToken?.claims?.Groups?.filter(group => group.includes("Staff"));
-	return global.structuredClone(modTableSettings[mod][table]);
+	return global.structuredClone(modTableSettings[modStaffList.includes(mod) ? mod : 'DefaultStaff'][table]);
```

Comment:
@kthorat-prog I would suggest looking for modTableSettings[mod] ? modTableSettings[mod] : modTableSettings['Default'] this way there is no hardcoded list.

Reply:
Also if another option needs to be added to the modTableSettings someone doesn't have to know that it also needs to be added to this hardcoded list.

Reply:
@oblodgett Yes...thats correct. I have removed the hardcoded list.